docs/maps/search.asciidoc

```diff
@@ -1,6 +1,11 @@
 [role="xpack"]
 [[maps-search]]
-== Searching your data
+== Search the data in a map
```

Comment:
"Search the data in a map" and "Search data in a map" sound clunky but I am not sure how to say it any better. How about "Search across layers" or just "Search"?